src/quo2/components/list_items/account_list_card/component_spec.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.list-items.account-list-card.component-spec
+  (:require
+    [quo2.foundations.colors :as colors]
+    [test-helpers.component :as h]
+    [quo2.components.list-items.account-list-card.view :as account-list-card]))
+
+(def account-props
+  {:customization-color :purple
+   :size                32
+   :emoji               "🍑"
+   :type                :default
+   :name                "Tip to Vegas"
+   :address             "0x0ah...78b"})
+
+(h/describe "List items: account list card"
+  (h/test "Test background-color for State: default"
+    (h/render [account-list-card/view
+               {:account-props account-props
+                :network       :ethereum
+                :state         :default
+                :action        :none}])
+    (h/has-style (h/get-by-label-text :container)
```

Comment:
Do we gain much by checking what colors/stlyes are in place in these tests? 🤔 
imo this might be too tightly coupling to the implementation and just makes it painful to change the styling. Visual tests (some day 🙏 ) handle this better as it just takes the color rendered etc.

I'm not saying we should remove these tests, but perhaps let's take this discussion first and then see whether we should continue adding test coverage for styling to components.
  
cc: @cammellos  @ulisesmac @briansztamfater @ilmotta

Reply:
@J-Son89, @OmarBasem My personal preference would be to only test styles maps if, for some reason, they are more complex to be computed, for example, with lots of conditions in a `cond`. Here I see that @OmarBasem wanted to make sure the color is correct for a given `state` property, which I think is something I would test myself too. Seems reasonable.

If the test was just verifying the value of a static/hardcoded style property, then I would say the test is almost useless or has negative value (since it needs to be maintained).

-----

Side note: I think `has-style` should convert the map argument to kebab case. I don't know if in this PR, but it's easy. All functions in the test helpers were written to help us write idiomatic Clojure. Now with the latest ClojureScript in our repo, we can just use `update-keys` https://cljs.github.io/api/cljs.core/update-keys

Reply:
@J-Son89 @ilmotta @OmarBasem 

I wouldn't test the background-color. For this kind of tests I think is better to check if something is rendered based on the props/state of the component without including styling, since we are testing styles through design reviews.

In this case, I think testing bg color is the same as testing any other style property that changes according to state, IMO, if we do tests in this way, we will end up having many many test and any style update will also need a test update, so I think it seems just like redundant information.

Reply:
After reading your comment @ulisesmac, I think the core of the problem here is the unit under test in this PR is questionable, so let's step back a bit. Quite an interesting discussion to have.

One of the advantages of having a separate style namespace is that it is decoupled from Reagent and React, and that makes most of its functions highly testable. But I don't mean any test, I mean the boring and reliable unit test.

So I see things differently now, along these lines:

1) If you do want to test certain styles are computed correctly, write unit tests in a file named `style.test.cljs`, not in the `component.spec.cljs` file. Unit tests are simpler, faster and REPL-friendly, contrary to component specs.
2) Let's give devs the freedom to choose if they want to test style functions or not. Quality is a spectrum, we don't need to make a guideline for either testing style functions or not testing them at all. After all, they are just pure functions for the most part, and if there's logic the dev wants to automatically check, then fine.
3) But having said those points, **avoid testing style functions in general**, unless you think the behavior should be better ironed out by automation. But again, let's not take this freedom from devs.
  
> since we are testing styles through design reviews.

Keep in mind that this sort of argument has been used to justify not testing anything in status-mobile because we have QAs 🤷🏼 Both the automated tests and design reviews are orthogonal to each other, and complementary.

Reply:
> avoid testing style functions in general, unless you think the behavior should be better ironed out by automation.

Completely agree!

> Keep in mind that this sort of argument has been used to justify not testing anything in status-mobile because we have QAs 🤷🏼 Both the automated tests and design reviews are orthogonal to each other, and complementary.

Yes, I agree too :smiley:

Reply:
Thanks for your input guys @ilmotta @ulisesmac @J-Son89 !

I agree testing styles may not be very useful (in most cases) and can be harder to maintain in the future. I have removed the styling tests 👍 

> Side note: I think has-style should convert the map argument to kebab case.
@ilmotta oh cool I wasn't aware of that